pkg/aggregator/internal/limiter/limiter.go

```diff
@@ -27,10 +29,17 @@ type Limiter struct {
 	keyTagName        string
 	telemetryTagNames []string
 	limit             int
+	global            int // global limit
+	current           int // sum(usage[*].current)
 	usage             map[string]*entry
+
+	// epoch, maxAge and lastSeen ensure eventual removal of entries that created an entry, but were
+	// never able to create contexts due to the global limit.
+	epoch  int
```

Comment:
```suggestion
	currentCycleAge  int
```

or something similar, I was (mistakenly) expecting `epoch` to be a unix epoch timestamp

Reply:
I was thinking about epoch-based reclamation algorithms when picking this name, but perhaps it's too niche. Do names in https://github.com/datadog/datadog-agent/blob/cd746c6bcacac102cd213b7a56224a76064ed9a3/pkg/aggregator/context_resolver.go#L246-L248 look better to you? I could use the same vocabulary here.